src/pages/lib-v1/auth/social/q/platform/[platform].mdx

```diff
@@ -247,193 +251,18 @@ const updatedAwsConfig = {
 Amplify.configure(updatedAwsConfig);
 ```
 
-### Full Sample
```

Comment:
Removed the full example as it wasn't adding much that wasn't covered in the other examples.